app/views/foreman/smart_proxies/_content_sync.html.erb

```diff
@@ -14,6 +14,10 @@
     </a>
   </div>
 
+  <div ng-hide="syncStatus.types_to_skip.length == 0">
+    <span translate>Pulp plugin missing for synchronizable types:</span> {{ syncStatus.types_to_skip.join(", ") }}.
```

Comment:
I think the user might want to know what this implies - and I think the answer is that these content types will be ignored/skipped when syncing